frameworks/g-cloud-10/questions/services/browsersSupported.yml

```diff
@@ -15,8 +15,10 @@ options:
     value: ie8
   - label: Internet Explorer 9
     value: ie9
-  - label: Internet Explorer 10+
-    value: ie10+
+  - label: Internet Explorer 10
+    value: ie10
+  - label: Internet Explorer 11
```

Comment:
Should this be `Internet Explorer 11+`

Reply:
I think you are correct, I will change that :)

Reply:
Yep, that's true. As you were.